src/usbasp.c

```diff
@@ -62,6 +62,8 @@
 # endif
 #endif
 
+#include <sys/time.h>
```

Comment:
Why was it needed to move `#include <sys/time.h>`  from line 34 to here?

Reply:
There's probably no good reason to move this. That's my guess...

Reply:
It is not related to this PR but rather related to MSVC build. So yes it is out of scope for this PR but it is nice to have.
* https://github.com/avrdudes/avrdude/issues/1211#issuecomment-1341820882